src/compass/defaults/s1_cslc_geo.yaml

```diff
@@ -17,6 +17,8 @@ runconfig:
       dynamic_ancillary_file_group:
           # Digital elevation model
           dem_file:
+          # IONEX file for ionosphere correction
+          ionex_file:
```

Comment:
Ionex is a file format, let's rename this to something more descriptive e.g. `tec_file `. Also let's add documentation of what this file is e.g. GNSS total electron content file in Ionex format